src/Features/Admin/Users/epics.js

```diff
@@ -0,0 +1,127 @@
+import { ofType } from 'redux-observable';
+import { of } from 'rxjs';
+import { ajax } from 'rxjs/ajax';
+import { catchError, mergeMap } from 'rxjs/operators';
+
+import { baseUrl } from '../../../scripts/constants';
+import { CREATE_USER, DELETE_USER_BY_ID, FETCH_USER_BY_ID, UPDATE_USER_BY_ID } from './ActionTypes';
+
+export const createUserEpic = (action$, state) =>
```

Comment:
WE can do create and update user request with react query if data is not needed on global level .. So try to use mutations from react query if data is not needed to be global ..